modules/serverless-creating-integrationsink-aws-sqs.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/event-sinks/serverless-integrationsink.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-creating-integrationsink-aws-sqs_{context}"]
+= Creating an IntegrationSink for AWS SNS
+
+You can create an `IntegrationSink` API resource to send CloudEvents to an Amazon SQS queue by applying a YAML manifest.
+
+.Prerequisites
+
+* You have created your AWS credentials and stored them in a Kubernetes Secret in the same namespace as the resource.
+* You have the OpenShift CLI (oc) installed and are logged in to the cluster.
+* You know the namespace where the `IntegrationSink` resource will be created.
+* A Knative broker or another event source exists to produce CloudEvents that will be delivered to this sink.
+
+.Procedure
+
+. Save the following YAML manifest as `integration-sink-aws-sqs.yaml`:
++
+[source,yaml]
+----
+apiVersion: sinks.knative.dev/v1alpha1
+kind: IntegrationSink
+metadata:
+  name: integration-sink-aws-sqs
+  namespace: knative-samples
+spec:
+  aws:
+    sqs:
+      arn: "arn:aws:s3:::my-queue"
```

Comment:
it should be SQS ARN, not s3 - the format is `arn:aws:sqs:<region>:<account>:my-queue`.

region used below is eu-north-1